actions/issues.go

```diff
@@ -37,10 +46,38 @@ func (v IssuesResource) ListOpen(c buffalo.Context) error {
 			requestParamToQueryFilter(&whereClause, &param, &filter)
 		}
 	}
-	// Retrieve all Issues from the DB
-	if err := q.Where(whereClause).All(issues); err != nil {
-		return errors.WithStack(err)
+
+	page := params.Get("page")
+	redisKey := "issues:" + whereClause + "and page=" + page
+	ok, err := cache.Exists(&cacheConn, redisKey)
+	if err != nil {
+		fmt.Println(errors.WithMessage(err, "Cache operation failed"))
```

Comment:
If smth goes wrong here, could you fetch the issues from the db instead of only printing an error?